src/packages/src/xroad/ubuntu/generic/xroad-center.postinst

```diff
@@ -219,10 +228,20 @@ configure | reconfigure)
     chmod 755 /var/lib/xroad/import
     chown xroad:xroad /var/lib/xroad/import
 
-    invoke-rc.d --quiet rsyslog try-restart || true
-    invoke-rc.d --quiet nginx try-restart || true
-    invoke-rc.d --quiet xroad-signer try-restart || true
-    invoke-rc.d --quiet xroad-jetty restart || true
+    RET=
+    db_get xroad-common/skip-cs-db-migrations || true
+    skip_migrations="${RET}"
+
+    if [ "$skip_migrations" = false ] ; then
+      invoke-rc.d --quiet rsyslog try-restart || true
+      invoke-rc.d --quiet nginx try-restart || true
+      invoke-rc.d --quiet xroad-signer try-restart || true
```

Comment:
I think these services should be restarted even if migrations are skipped.

Reply:
Ok, updated in https://github.com/nordic-institute/X-Road/pull/1024/commits/e8388d96102d41e86c96773b060c6e647413ce4f